.changeset/silver-groups-shout.md

```diff
@@ -0,0 +1,45 @@
+---
+"@biomejs/biome": minor
+---
+
+Added support for formatting `.html` files. The formatting is considered **experimental,** and it's only opt-in via configuration:
+
+```json
+{
+  "html": {
+    "formatter": {
+      "enabled": true
+    }
+  }
+}
+```
+
+Biome formatter attempts to format as Prettier, however some default options might differ.
+
+An option `html.formatter.selfCloseVoidElements` allows to control if the trailing `/` of [void elements](https://html.spec.whatwg.org/#void-elements).
+
+**By default**, Biome formatter will *remove* the `/`:
+
+```diff
+- <input />
++ <input >
```

Comment:
I assume the formatter would/should trim the space before it too, no?

```suggestion
+ <input>
```

Reply:
As for the current implementation, no, as the tests should. I can follow up with another PR to change that, and update the changeset too